components/event/EventCard.js

```diff
@@ -137,6 +138,15 @@ export default function EventCard({ manage, event, usernames }) {
           )}
         </div>
       </div>
+      {event.tags?.length > 0 &&
+        event.tags.map((tag, index) => {
+          const trimmedTag = tag.trim();
+          if (!trimmedTag) {
+            return null;
+          }
+
+          return <Tag name={trimmedTag} key={index} />;
```

Comment:
The tag component is expecting an `onClick` event, so has a hover state

<img width="1332" alt="Screenshot 2023-10-15 at 04 32 02" src="https://github.com/EddieHubCommunity/BioDrop/assets/624760/5bedb8b3-2489-4499-9889-4f8411e44edb">

Using the tag simple component I think would be better here, later if we add an on click event we can change it back

<img width="1409" alt="Screenshot 2023-10-15 at 04 37 27" src="https://github.com/EddieHubCommunity/BioDrop/assets/624760/76bd11d0-8e47-4203-9fa7-057d0a3519d6">